aion/web/elm/src/Update.elm

```diff
@@ -269,6 +307,32 @@ update msg model =
                 else
                     categoryFormValidationErrorToast (model ! [])
 
+        CreateNewRoom ->
+            let
+                roomForm =
+                    model.panelData.roomForm
+
+                validationErrors =
+                    []
+
+                subjectIds =
+                    List.map (\( x, _ ) -> x) (Multiselect.getSelectedValues model.panelData.categoryMultiSelect)
+            in
+                if List.isEmpty validationErrors then
+                    ( model, createRoom model.panelData.roomForm subjectIds )
+                else
+                    roomFormValidationErrorToast (model ! [])
+
+        MultiselectMsg subMsg ->
```

Comment:
What's the reason for the sub prefix in subMsg subCmd and subModel? I'm curious, perhaps there is a better name for that.

Reply:
I think it is because its subscription related update.